bluesky_queueserver/manager/plan_queue_ops.py

```diff
@@ -759,16 +758,16 @@ async def _add_plan_to_history(self, plan):
         int
             The new size of the history.
         """
-        history_size = await self._r_pool.rpush(self._name_plan_history, json.dumps(plan))
+        history_size = await self._r_pool.rpush(self._name_plan_history, json.dumps(item))
         return history_size
 
-    async def _get_plan_history_size(self):
+    async def _get_history_size(self):
         """
-        See ``self.get_plan_history_size()`` method.
+        See ``self.get_history_size()`` method.
         """
         return await self._r_pool.llen(self._name_plan_history)
 
-    async def get_plan_history_size(self):
+    async def get_history_size(self):
         """
         Get the number of items in the plan history.
```

Comment:
```suggestion
        Get the number of items in the items history.
```

Reply:
I think that we will still use 'plan queue' and 'plan history'. They are mostly be used for plans anyway, so the name makes sense. But they will contain 'items'.